pom.xml

```diff
@@ -26,7 +26,7 @@
 		<redis.lettuce.version>5.2.2.RELEASE</redis.lettuce.version>
 		<opensrp.updatePolicy>always</opensrp.updatePolicy>
 		<nexus-staging-maven-plugin.version>1.5.1</nexus-staging-maven-plugin.version>
-		<opensrp.core.version>2.4.3-SNAPSHOT</opensrp.core.version>
+		<opensrp.core.version>2.4.5-SNAPSHOT</opensrp.core.version>
```

Comment:
Should this be 2.4.5 or 2.4.4

Reply:
The `server-core` snapshot version published that introduces the domain changes is `v2.4.6`